### PR TITLE
Scenes: Set min-height of left panel editor panes to 0

### PR DESCRIPTION
**Pull request by kaydelaney**
Fixes an issue where the query pane couldn't be made smaller than it's min-content height.

### CODE REVIEW

File: public/app/features/dashboard-scene/panel-edit/PanelDataPane/PanelDataPane.tsx [hunk 1]
```diff
@@ -175,6 +175,13 @@ function getStyles(theme: GrafanaTheme2) {
...
       borderBottom: 'none',
       borderTopRightRadius: theme.shape.radius.default,
+      flexGrow: 1,
+    }),
+    tabsBar: css({
+      flexShrink: 0,
+    }),
+    scroll: css({
+      background: theme.colors.background.primary,
     }),
```
Comment (dprokop) [inline public/app/features/dashboard-scene/panel-edit/PanelDataPane/PanelDataPane.tsx:185]: Why is this needed?

**Review by dprokop — APPROVED** [review]
Works good now:)

File: public/app/features/dashboard-scene/panel-edit/PanelDataPane/PanelDataPane.tsx (hunk 1 above)
Comment (torkelo) [reply public/app/features/dashboard-scene/panel-edit/PanelDataPane/PanelDataPane.tsx:185]: I tested without it and could not see any change, there TabContent has a background set

**Review by torkelo — APPROVED** [review]
beyond the scroll background css looks good

File: public/app/features/dashboard-scene/panel-edit/PanelDataPane/PanelDataPane.tsx (hunk 1 above)
Comment (kaydelaney) [reply public/app/features/dashboard-scene/panel-edit/PanelDataPane/PanelDataPane.tsx:185]: With:

https://github.com/grafana/grafana/assets/45561153/5862943a-3b40-4912-9304-22d3cb6d7c65

Without:


https://github.com/grafana/grafana/assets/45561153/a234feb4-2f74-494f-8d56-5e6f912ade82

**Comment by kaydelaney** [comment]
> beyond the scroll background css looks good

See https://github.com/grafana/grafana/pull/80641#discussion_r1457279730
